index.js

```diff
@@ -156,26 +178,42 @@ const ScrollableTabView = React.createClass({
          }}
          ref={(scrollView) => { this.scrollView = scrollView; }}
          {...this.props.contentProps}>
-         {this._children().map((child, idx) => {
-           return <View
-             key={child.props.tabLabel + '_' + idx}
-             style={{width: this.state.containerWidth, }}>
-             {child}
-           </View>;
-         })}
+           {scenes}
         </ViewPagerAndroid>
       );
     }
   },
 
+  _composeScenes() {
+    return this._children().map((child, idx) => {
+      let key = this._makeSceneKey(child, idx);
+      return (
+        <SceneComponent
+          key={key}
+          style={{width: this.state.containerWidth, }}
+          selected={(this.state.currentPage === idx)}>
```

Comment:
https://github.com/Khan/style-guides/blob/master/style/react.md#align-and-sort-html-properties